test/VM_Test/src/com/ibm/j9/recreateclass/tests/RecreateClassCompareTest.java

```diff
@@ -129,10 +129,11 @@ public void testStackMapTable() throws Exception {
 
 	public void testStackMapTableTestWithBooleanArray() throws Exception {
 		System.out.println("Start testStackMapTableTestWithBooleanArray");
-		/* Recreated StackMapTableWithByteBooleanArrayTest is expected to have byte array instead of boolean array 
-		 * in verification_type array of StackMapTable.
+		/* To follow the Java 9 Spec in which boolean arrays are different from byte arrays
```

Comment:
Does this test still pass on Java 8 with this change?

Reply:
I verified the updated test on Java 9 & Java 8 and both passed without any issue.